OpenOversight/app/main/views.py

```diff
@@ -456,8 +470,8 @@ def add_salary(officer_id):
 @login_required
 @ac_or_admin_required
 def edit_salary(officer_id, salary_id):
+    officer = Officer.query.filter_by(id=officer_id).one()
```

Comment:
This had to be moved up so I could access the `officer` variable in the cache entry removal function.